extension/scripts/publish.sh

```diff
@@ -3,6 +3,7 @@ COMMIT_MESSAGE=$(git log --format=%B -n 1)
 if [[ $COMMIT_MESSAGE == "release-"* || $COMMIT_MESSAGE == "[Simsala] automatic release"* ]]
 then
     echo "Publishing"
+    git config pull.rebase false
```

Comment:
```suggestion
```

I think that is the default

Reply:
yeah, but we get the warning logs, which are quite annoying and recommend doing this. Better leave this here I think